test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5560,11 +5571,22 @@ def revert_to_defaults(self, reverthooks=True, revertqueues=True,
             self.manager(MGR_CMD_CREATE, QUEUE, a, id='workq', expect=True)
             setdict.update({ATTR_dfltque: 'workq'})
         if reverthooks:
-            hooks = self.status(HOOK, level=logging.DEBUG)
-            hooks = [h['id'] for h in hooks]
-            a = {ATTR_enable: 'false'}
-            if len(hooks) > 0:
-                self.manager(MGR_CMD_SET, MGR_OBJ_HOOK, a, hooks, expect=True)
+            if self.platform == 'cray' or self.platform == 'craysim':
+                if self.du.cmp(self.hostname, self.dflt_mpp_hook,
```

Comment:
I think this code should be reverted once we [PP-817](https://pbspro.atlassian.net/browse/PP-817) get complete. @borlesanket Can you please add a comment on [PP-817](https://pbspro.atlassian.net/browse/PP-817) for same?